ML_Embedded_Evaluation_Kit.md

```diff
@@ -443,33 +446,45 @@ If you followed Option-1 to install Arm DS and Arm Clang Compiler for Embedded,
 Copy the converted binaries (`.bin` file) to the following directory: `/home/$USER/app-release-exec-linux/build/images`
 
 
-#### The next step is to generate the binary ATOC image for the applications.
-
-1. Create a new JSON file called ```\build\config\kws_demo.json``` with the following content.
-    ```
-    {
-        "HE_Voice": {
-            "binary": "ethos-u-alif_kws.bin",
-            "version": "1.0.0",
-            "mramAddress": "0x80480000",
-            "cpu_id": "M55_HE",
-            "flags": ["boot"],
-            "signed": false
-        }
-    }
-    ```
+### 2. Generate the binary ATOC image for the applications
+
+1. Create a new JSON file called ```\build\config\kws_demo.json``` with the following content. (You can find some example .json files from this repository.)
+	```
+	{
+		"HE_Voice": {
+			"binary": "ethos-u-alif_kws.bin",
+			"version": "1.0.0",
+			"mramAddress": "0x80480000",
+			"cpu_id": "M55_HE",
+			"flags": ["boot"],
+			"signed": false
+		},
+		"DEVICE": {
+			"disabled" : false,
+			"binary": "app-device-config.json",
```

Comment:
DevKit-e1c is also supported. Maybe add note after the chapter to tell user that with DevKit-e1c you need to use app-device-config-1c.json